usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -77,6 +77,7 @@ This endpoint returns a list of data that is associated with the award profile p
                     "multi_year_contract": "N",
                     "consolidated_contract": "N",
                     "type_of_contract_pricing": "J",
+                    "national_interest_action": "NONE",
```

Comment:
Is there any reason this isn't paired with the `national_interest_action_description` below?

Reply:
I copied the actual API JSON response into the markdown

Reply:
Moved